scripts/storybook/src/utils.js

```diff
@@ -170,6 +170,13 @@ function _createCodesandboxRule(allPackageInfo = getAllPackageInfo()) {
 
       return acc;
     }, /** @type import('storybook-addon-export-to-codesandbox').BabelPluginOptions*/ ({}));
+
+    return {
+      ...importMappings,
+      '@fluentui/react-data-grid-react-window': {
```

Comment:
we should come up with something more robust, as hardcoding this mapping for every extension package will not scale

Reply:
Tracking in #26691